src/Components/Admin/Postulants/Form/index.js

```diff
@@ -362,403 +232,660 @@ function PostulantsForm() {
           ? contactToValue
           : setShowError('Hours must have HH:MM format')
       },
-      studies: studiesBodyConstructor(),
-      workExperience: workExperienceBodyConstructor()
+      studies: studiesBodyConstructor(formValues),
+      workExperience: workExperienceBodyConstructor(formValues)
     };
 
     if (postulantId) {
-      dispatch(updatePostulant(postulantId, postulant)).then((response) => {
+      dispatch(updatePostulant(postulantId, body)).then((response) => {
         if (response) {
           history.push('/admin/postulants/list');
         }
       });
     } else {
-      dispatch(addPostulant(postulant)).then((response) => {
+      dispatch(addPostulant(body)).then((response) => {
         if (response) {
           history.push('admin/postulants/list');
         }
       });
     }
   };
 
+  const validate = (formValues) => {
+    const errors = {};
+    // First Name
+    if (!formValues.firstName) {
+      errors.firstName = 'First Name is required';
+    }
+    if (!formValues.firstName?.match(/^[a-zA-Z]+$/)) {
```

Comment:
When typing  "Del Moro" in Last Name input I recieve the error message "Last name must contain only letters"

Reply:
Yes, you're right. Done